clients/src/main/java/org/apache/kafka/clients/NodeApiVersions.java

```diff
@@ -67,8 +66,7 @@ public static NodeApiVersions create(Collection<ApiVersion> overrides) {
                 }
             }
             if (!exists) {
-                apiVersions.add(new ApiVersion(apiKey.id, ProtoUtils.oldestVersion(apiKey.id),
-                    ProtoUtils.latestVersion(apiKey.id)));
+                apiVersions.add(new ApiVersion(apiKey.id, apiKey.oldestVersion(), apiKey.latestVersion()));
```

Comment:
Maybe we could add a constructor to `ApiVersion` which accepts an instance of `ApiKeys`?